lessons/AI/ai-statistical-lang-models/langs/en-us/index.adoc

```diff
@@ -0,0 +1,452 @@
+= Statistical Language Modeling
+
+@description{Students consider how statistics and probability drive AI text generation, learning that language models can produce text that sounds credible but may not actually be credible.}
+
+@ifproglang{pyret}{
+@lesson-prereqs{ai-training}
+}
+
+@keywords{}
+
+[@lesson-intro-table]
+|===
+| Lesson Goals
+| Students will be able to...
+
+@objectives
+
+| Student-facing Lesson Goals
+|
+
+- Let's explore how probability influences the output of generative AI models.
+
+
+| Materials
+|[.materials-links]
+@material-links
+
+|===
+
+
+
+== Introduction to Statistical Language Modeling
+
+@objective{predictive-text}
+
+=== Overview
+
+Students explore how text prediction relies on statistical language modeling.
+
+=== Launch
+
+While texting, emailing, or even conducting a web search, you've probably noticed that your thoughts and sentences are sometimes finished for you! Predictive text is now a common feature of many apps, and it is made possible by a certain form of AI: the statistical language model.
+
+@lesson-instruction{
+- Consider this start of a phrase: "Close the..."
+- On a piece of paper, write down the next word that pops into your mind.
+- Write down two additional next words.
+- As a class, we will generate a list of all of the words we suggested.
+}
+
+@teacher{
+Students will think of words like "door", "box", "microwave", "laptop"... On the board, record their word suggestions. If a word is repeated, keep a tally of how many students suggest each particular word.
+}
+
+=== Investigate
+
+Let's look a little more closely at the words we generated.
+
+@QandA{
+@Q{Which words had the _most_ tally marks?}
+@Q{Which words had the _least_ tally marks?}
+@Q{Are you surprised by the most and least common words? Why or why not?}
+@Q{What was our class probability of proposing each of our top three words?}
+@A{For example: If there are 25 students in your class, and 8 students chose “door”, the probability of a student choosing that word is 8/25.}
+}
```

Comment:
@shriram This probability feels very wrong to me, but I am not an expert. Emmanuel says that we can trust you to have gotten it right... Please confirm that you feel comfortable with the wording of this question and its answer.